docs/concepts/jcloud/configuration.md

```diff
@@ -226,12 +226,13 @@ executors:
+| max              | 2           | int, up to 5                                | Maximum number of replicas                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     |
+| metric           | concurrency | `concurrency`  /   `rps` / `cpu` / `memory` | Metric for scaling                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             |
+| scale_down_delay | 30s         | str, 0s <= value <= 1h                      | Time window which must pass at reduced concurrency before a scaling down                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       |
+| target           | 100         | int                                         | Target number the replicas try to maintain. If metric is `cpu`, this is the % CPU utilization (e.g. 60 means replicas will be scaled when pods on average reach 60% CPU utilization). If metric is `memory`, this is the target millibytes of memory (e.g. 200m means replicas will be scaled when each pod memory consumption exceeds `200m` on average). If metric is `rps`, this is the target number of total requests processed per second per replica. Lastly, if metric is `concurrency`, this is the target number of concurrent requests processed at any given time. |
```

Comment:
we should elabroate the metrics target in below, not put too much information in the table.